fuzz/clean.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+rm -r radamsa
+rm -r 0*
+rm -r 1*
+rm -r 2*
+rm -r 3*
+rm -r 4*
+rm -r 5*
+rm -r 6*
+rm -r 7*
+rm -r 8*
+rm -r 9*
```

Comment:
If you prefix all folders which contain information about fail with e.g. `fail-`, you can delete just `rm -r ./fail-*`.